test/influxdb_v2/deleter/predicate_test.exs

```diff
@@ -0,0 +1,128 @@
+defmodule Instream.InfluxDBv2.Writer.PredicateTest do
+  use ExUnit.Case, async: true
+
+  @moduletag :"influxdb_include_2.x"
+
+  alias Instream.TestHelpers.TestConnection
+
+  defmodule EmptyTagSeries do
+    use Instream.Series
+
+    series do
+      measurement "empty_tags"
```

Comment:
The `measurement` value is the reason for the flaky tests.

The `Instream.InfluxDBv2.Writer.LineTest` uses the same measurement but expects exactly one specific value to be written. If the deleter test runs before that there are two values and the tests fail.

If you pass the seed using `mix test --seed 926560` (one of the failing orders) you should have a reproducible case. Changing the `measurement` to a value not used in other tests should be enough :)

Reply:
Yes, you are right. Thank you it's fixed.